src/main/java/com/andrzejewski/todolist/task/TaskRepository.java

```diff
@@ -0,0 +1,23 @@
+package com.andrzejewski.todolist.task;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public interface TaskRepository extends CrudRepository<TaskEntity, Long> {
+    List<TaskEntity> findAll();
+
+    default List<TaskEntity> findAllTasksByUserId(Long userId) {
+        List<TaskEntity> taskEntities = findAll();
+
+        List<TaskEntity> usersTaskEntities = new ArrayList<>();
+
+        for (TaskEntity taskEntity : taskEntities) {
+            if (taskEntity.getUserId().equals(userId)) usersTaskEntities.add(taskEntity);
+        }
+        return usersTaskEntities;
+    }
```

Comment:
This is not what i meant.
This operation should be done via the SQL query, not in java code, for example
```SQL
SELECT * FROM tasks WHERE user_id = 1;
```
You should create a repository method that when executed executes the query and returns the result. This is possible with Spring Data JPA. In order to do it, declare the method signature and then:
- use `@Query` to specify the SQL or JPQL query to execute
- allow Spring Data JPA to infer the query from method name

For more info refer to the [docs](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods).